docs/futures/define-custom-roles.rst

```diff
@@ -125,4 +132,96 @@ Your topology file could look like this:
             group: "foo"
         bar:
           - principal: "User:bandana"
-            group: "bar"
\ No newline at end of file
+            group: "bar"
+
+More generic usage of roles
+-----------
+
+Lets assume you need to deploy multiple instances of an application that is not willing to use JulieOps topic hierarchy,
+requires multiple topics and custom ACLs. You could define a role for each instance, or for each group/topic,
+but that would clutter the roles file quickly. You would however like to have these topics in JulieOps, for ACL management
+and for documentation. For example Kafka MirrorMaker needs many topics which would be nice to group together in JulieOps
+config.
+
+For brevity example below gives too much permissions (ALL), but works as example how feature works.
+
+.. code-block:: YAML
+
+    roles:
+      - name: "mirrorMaker"
+        acls:
+          - resourceType: "Topic"
+            resourceName: "{{statusTopic}}"
+            patternType: "LITERAL"
+            host: "*"
+            operation: "ALL"
+            permissionType: "ALLOW"
```

Comment:
Would it make sense to make `permissionType: ALLOW` the default, or do we want to be very explicit here?

Reply:
It would. And change is small. Added to PR.